realm/realm-library/src/main/java/io/realm/util/Pair.java

```diff
@@ -14,19 +14,22 @@
  * limitations under the License.
  */
 
-package io.realm.internal.util;
+package io.realm.util;
```

Comment:
Not sure I understand why we need to move other existing classes? Which ones?
Also, the Javadoc should be generated for this package, we just exclude `internal`:
```
    exclude '**/internal/**'
    exclude '**/BuildConfig.java'
    exclude '**/R.java'
```